src/struct_ls/red_black_constantcoef_gs.c

```diff
@@ -356,12 +333,12 @@ hypre_RedBlackConstantCoefGS( void               *relax_vdata,
                         {
                            xp[xi] =
                               (bp[bi] -
-                               App0 * xp[xi + xoff0] -
```

Comment:
Why not App0[Ai]?

Reply:
In this branch we just have the diagonal coefficient as variable, while the others are constant. So, all the `App`s have to be accessed via `App[0]`